src/List/List.js

```diff
@@ -45,13 +49,16 @@ export default class List extends Component {
       className: classNameProp,
       component,
       padding,
+      children,
       ...other,
     } = this.props;
     const classes = this.context.styleManager.render(styleSheet);
+    const firstChild = Children.toArray(children)[0];
```

Comment:
Kind of how it's done here https://github.com/callemall/material-ui/blob/next/src/List/ListItem.js#L100-L102.